src/vs/base/common/codicons.ts

```diff
@@ -13,582 +13,607 @@ export function getCodiconAriaLabel(text: string | undefined) {
 	return text.replace(/\$\((.*?)\)/g, (_match, codiconName) => ` ${codiconName} `).trim();
 }
 
+export interface ICodicon extends CSSIcon {
+	readonly codiconFontCharacter: string;
```

Comment:
@aeschli fyi I decided here to name `fontCharacter` actually `codiconFontCharacter` to be able to implement a good `isCodicon` method that can check on that. Previously we were able to do `instanceof Codicon`, but with this change we no longer can. If we kept `fontCharacter`, there is a chance we would wrongly match on an object that has both `id` and `fontCharacter`. Since I found a few more types with `fontCharacter` I decided to go this path, but please verify that is OK.